### ISSUE TITLE
Feature requests. Fading between images and load images before they are shown

### ISSUE THREAD

**Issue by TheLogan** [lokesh/lightbox2#321]
Hi, I've just installed Lightbox on the website I'm currently writing, and I really really like it so far. But there are 2 features I'd love to request that I can't find (so I assume they aren't already there).
I'm fairly new to web development, so I'll try to explain what I mean as best I can, but I'll probably get a few technical terms wrong, bear with me xD (that also means that I don't know if they are doable...)

Feature 1 I think the correct term would be cross-fading?
Instead of fading image 'a' out, and then fade image 'b' in when a is gone, I would much prefer fading image 'a' out and simultaneously fade image 'b' in, so it doesn't keep fading to the white background.
I know there are probably a few oddities when fading between images of different sizes, but if possible what would look best is just to crop the images so they stay the same size.
Personally I get slightly annoyed when I brows galleries, and the images keep fading to the background, and  I'm probably not alone in this... =)

Feature 2 Pre-load? Cache? something like that anyway =P
Basically if I click image 'f' image 'g' and 'd' should be loaded secondarily after image 'f' has loaded. That way, when you're browsing the gallery the chances of having to wait for an image to load is slimmer. (Is this perhaps something that the rest of the website has to do, rather than lightbox..?)

Anyway, I hope this is doable and something you'd be interested in trying. I'd do it my self, but I'm still very much a jQuery noob (though I am going through tutorials, trying to learn), and besides, I'm probably not the only one who would find it useful. =)

**Comment by lokesh** [comment]
Closing out old, inactive issues.

Re: Feature 1
I'm looking into adding animated transition between images. I'm particularly curious about panning that is triggered by touch based swiping gestures.

Re: Feature 2
When viewing a set of images the next image is preloaded behind the scenes.
